pallets/attestation/src/mock.rs

```diff
@@ -217,13 +214,12 @@ pub(crate) mod runtime {
 	impl frame_system::Config for Test {
 		type RuntimeOrigin = RuntimeOrigin;
 		type RuntimeCall = RuntimeCall;
-		type Index = u64;
-		type BlockNumber = u64;
 		type Hash = Hash;
 		type Hashing = BlakeTwo256;
 		type AccountId = AccountId;
 		type Lookup = IdentityLookup<Self::AccountId>;
-		type Header = Header;
+		type Block = Block;
+		type Nonce = u32;
```

Comment:
This was `u64`, right?

Reply:
I can not remember why I changed the type… So I reverted it: [74a1a67](https://github.com/KILTprotocol/kilt-node/pull/559/commits/74a1a674b19b7e9ebe95d1832d8ef41fbeb3b8e1)